src/groovy/org/pih/warehouse/core/Constants.groovy

```diff
@@ -34,6 +34,8 @@ class Constants {
     static final String EXPIRATION_DATE_FORMAT = "MM/dd/yyyy"
     static final String DELIVERY_DATE_FORMAT = "MM/dd/yyyy HH:mm XXX"
     static final String EUROPEAN_DATE_FORMAT = "dd/MM/yyyy"
+    static final String TABLE_MONTH_YEAR_DATE_FORMAT = "MMM DD, yyyy"
+    static final String TABLE_DATE_DEFAULT_VALUE = "-"
```

Comment:
Something is bothering me about the use of "TABLE" here. I understand the need to add context and I'm aware that naming is hard. But TABLE seems arbitrary since we might actually display this outside of a table. So let's try to come up something generic, but not restricting. I think DISPLAY_DATE_FORMAT or DEFAULT_DISPLAY_DATE_FORMAT would be fine for now.